samples/unit-tests/coloraxis/coloraxis-title/demo.js

```diff
@@ -9,10 +9,55 @@ QUnit.test('Color axis title should render', function (assert) {
         }]
     });
 
-    var colorAxis = chart.colorAxis[0];
+    const colorAxis = chart.colorAxis[0];
     assert.ok(
         colorAxis.axisTitle &&
         colorAxis.axisTitle.element.textContent === 'Test Title',
         'Color axis title is rendered with correct text'
     );
-});
\ No newline at end of file
+});
+
+QUnit.test(
+    'Color axis title should rotate with vertical legend',
+    function (assert) {
+        // Reuse the chart from the previous test by updating options
+        const chart = Highcharts.chart('container', {
+            legend: {
+                layout: 'vertical',
+                align: 'right',
+                verticalAlign: 'middle'
+            },
+            colorAxis: {
+                title: { text: 'Test Title' }
+            },
+            series: [{
+                type: 'heatmap',
+                data: [[0, 0, 1], [0, 1, 2], [1, 0, 3], [1, 1, 4]]
+            }]
+        });
+
+        const colorAxis = chart.colorAxis && chart.colorAxis[0];
+        const title = colorAxis && colorAxis.axisTitle;
+
+        assert.ok(title && title.element, 'Color axis title exists');
```

Comment:
The best assert's message is a one with "should" just to be sure that it described what should (or shouldn't) happen as the result of the test. Also, this one seems pointless when you have the next test that checks the title's text.